packages/data-mocked/src/iot-sitewise/handlers/batchGetAssetPropertyValue/batchGetAssetPropertyValueHandler.ts

```diff
@@ -81,8 +82,9 @@ export class AssetPropertyValueFactory {
       ...this.#createDefaults(),
       ...partialAssetPropertyValue,
       value: {
+        // nullValue: { valueType: 'D' },
```

Comment:
commented out on purpose? can it be deleted?

Reply:
I was leaving these in for future testing purposes.